### ISSUE TITLE
Setting accessibility properties on Views forces them to be focusable

### ISSUE THREAD

**Issue by jonthysell** [microsoft/react-native-windows#3874]
Accessibility properties set on Views get resolved by the DynamicAutomationPeer on ViewControl. So if you want to set accessibility properties, you have to have a control, which is now focusable. The inner ViewPanel does not have a peer, because if they both have peers then every View in the UIA tree appears twice.

We don't have the APIs exposed for external XAML control developers to delete/invalidate peers, which we need in order to support our complicated live ViewControl>Border>ViewPanel swapping.

Related issues: #3836

**Comment by chrisglein** [comment]
@jonthysell All accessibility properties or just some?
Is it not possible to make the Control not focusable? Like by setting [Focusable](https://docs.microsoft.com/en-us/dotnet/api/system.windows.uielement.focusable?view=netframework-4.8)?

**Comment by jonthysell** [comment]
> @jonthysell Jon Thysell (JAUNTY) FTE All accessibility properties or just some?
> Is it not possible to make the Control not focusable? Like by setting [Focusable](https://docs.microsoft.com/en-us/dotnet/api/system.windows.uielement.focusable?view=netframework-4.8)?

Only if we switch to WPF. :)

**Comment by acoates-ms** [comment]
Can't you set IsTabStop = false?

**Comment by NickGerleman** [comment]
Should be fixed since 0.64 with https://github.com/microsoft/react-native-windows/pull/6728
